ee/clickhouse/migrations/0025_remove_dollar_from_materialized_columns.py

```diff
@@ -0,0 +1,48 @@
+import structlog
+from clickhouse_driver.errors import ServerException
+from infi.clickhouse_orm import migrations
+
+from ee.clickhouse.client import sync_execute
+from ee.clickhouse.materialized_columns.columns import get_materialized_columns, materialized_column_name
+from posthog.constants import GROUP_TYPES_LIMIT
+from posthog.settings import CLICKHOUSE_CLUSTER, CLICKHOUSE_REPLICATION
+
+logger = structlog.get_logger(__name__)
+
+
+def rename_column(table, current_name, new_name):
+    if CLICKHOUSE_REPLICATION and table == "events":
+        sync_execute(
+            f"ALTER TABLE {table} RENAME COLUMN \"{current_name}\" TO {new_name} ON CLUSTER '{CLICKHOUSE_CLUSTER}'"
+        )
+        sync_execute(
+            f"ALTER TABLE sharded_{table} RENAME COLUMN \"{current_name}\" TO {new_name} ON CLUSTER '{CLICKHOUSE_CLUSTER}'"
+        )
+    else:
+        sync_execute(f'ALTER TABLE {table} RENAME COLUMN "{current_name}" TO {new_name}')
+    logger.info("Renamed column containing $ sign.", table=table, current_name=current_name, new_name=new_name)
+
+
+def rename_column_if_exists(table, current_name, new_name):
+    try:
+        rename_column(table, current_name, new_name)
+    except ServerException as err:
+        logger.info("Failed to rename column, ignoring.", table=table, current_name=current_name, new_name=new_name)
```

Comment:
This seems more like a warning rather than info. Also, can we capture the exception with `exc_info=True`.

Reply:
Will clear up the wording. This is safe to ignore as the column would already be renamed on a fresh install.